langchain/vectorstores/chroma.py

```diff
@@ -95,7 +95,7 @@ def __init__(
                 _client_settings = client_settings
             elif persist_directory:
                 _client_settings = chromadb.config.Settings(
-                    chroma_db_impl="duckdb+parquet",
+                    is_persistent=True,
```

Comment:
is update backwards compatible with old config settings? ie if someone was passing in
```python
_client_settings = chromadb.config.Settings(
                     chroma_db_impl="duckdb+parquet",
                     persist_directory=persist_directory,
                 )
```
directly before, would behavior they're seeing change?

Reply:
they will see this error

> You are using a deprecated configuration of Chroma. Please pip install chroma-migrate and run chroma-migrate to upgrade your configuration. See https://docs.trychroma.com/migration for more information or join our discord at https://discord.gg/8g5FESbj for help!

more here https://docs.trychroma.com/migration